src/sst/core/link.cc

```diff
@@ -41,10 +42,13 @@ Link::Link(LinkId_t id) :
     if ( uninitQueue == NULL )
 	uninitQueue = new UninitializedQueue("ERROR: Trying to send or recv from link during initialization.  Send and Recv cannot be called before setup.");
     if ( afterInitQueue == NULL )
```

Comment:
Checking `afterInitQueue`, but then assigning to `afterRunQueue`.  This doesn't feel right.

Reply:
Fixed.